src/components/auth/AuthForm.js

```diff
@@ -8,15 +8,14 @@ export const AuthForm = ({
   return (
     <>
       <form onSubmit={onSubmit}>
-        <div>
-          <label htmlFor="email">Email</label>
-          <input type="email" name="email" id="email" />
-        </div>
+        <InputAuthText value="Email" name="email" type="email" id="email" />
         {!isPasswordHidden && (
-          <div>
-            <label htmlFor="password">Password</label>
-            <input type="password" name="password" id="password" />
-          </div>
+          <InputAuthText
+            value="Password"
+            name="password"
+            type="password"
+            id="password"
+          />
```

Comment:
jw

Reply:
a ten nawias niepotrzebny chyba

Reply:
ten nawias to zamknięcie InputAuthText tylko przy zapisie prettier mi automatycznie go wywalił do innej linijki

Reply:
chodzi mi o ten okrągły nawias, nie o zamknięcie tagu :) zaczynający się w 12 linii a kończońcy w 19